critiquebrainz/ws/review/views.py

```diff
@@ -23,6 +23,15 @@ def review_entity_handler(review_id):
     :statuscode 404: review not found
 
     :resheader Content-Type: *application/json*
+
+    .. code-block:: bash
+
+       $ curl -X GET https://critiquebrainz.org/ws/1/review/b7575c23-13d5-4adc-ac09-2f55a647d3de
+
+    .. code-block:: json
+
+       {"review":{"created":"Tue, 10 Aug 2010 00:00:00 GMT","edits":0,"entity_id":"03e0a99c-3530-4e64-8f50-6592325c2082","entity_type":"release_group","id":"b7575c23-13d5-4adc-ac09-2f55a647d3de","language":"en","last_updated":"Tue, 10 Aug 2010 00:00:00 GMT","license":{"full_name":"Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported","id":"CC BY-NC-SA 3.0","info_url":"https://creativecommons.org/licenses/by-nc-sa/3.0/"},"rating":0,"source":"BBC","source_url":"http://www.bbc.co.uk/music/reviews/3vfd","text":"REVIEW GOES HERE","user":{"created":"Wed, 07 May 2014 14:55:23 GMT","display_name":"Paul Clarke","id":"f5857a65-1eb1-4574-8843-ae6195de16fa","karma":0,"user_type":"Noob"},"votes_negative":0,"votes_positive":0}}
```

Comment:
This is currently unreadable and needs to be formatted. See https://acousticbrainz.readthedocs.io/api.html#post--api-v1-datasets- for example.